lib/WP_Auth0_Settings_Section.php

```diff
@@ -53,7 +53,7 @@ public function init_menu() {
 
 			add_submenu_page( $main_menu, __( 'Help', 'wp-auth0' ), __( 'Help', 'wp-auth0' ), 'manage_options', 'wpa0-help', array( $this, 'redirect_to_help' ) );
 
-			add_submenu_page( $main_menu, __( 'Auth0 for WordPress - Setup Wizard', 'wp-auth0' ), __( 'Setup Wizard', 'wp-auth0' ), 'manage_options', 'wpa0-setup', array( $this->initial_setup, 'render_setup_page' ) );
+			add_submenu_page( null, __( 'Auth0 for WordPress - Setup Wizard', 'wp-auth0' ), __( 'Setup Wizard', 'wp-auth0' ), 'manage_options', 'wpa0-setup', array( $this->initial_setup, 'render_setup_page' ) );
```

Comment:
Removes the menu item when the plugin is already setup.

Reply:
But it's always `null` how can it be toggled ?

Reply:
This is in an `if` block that checks if the plugin is configured or not. If it's already configured, we don't want to show the wizard.

Reply:
ok expanded the view, so if it's not ready I can see it adds the `Setup Wizard`. 
So it must be already added via WP_Auth0_Admin.php? Hence the requirement to removal?